hw/ip/clkmgr/dv/cov/clkmgr_manual_cov_unr_excl.el

```diff
@@ -0,0 +1,49 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// This contains some obvious exclusions the UNR tool didn't flag.
+
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: maturana
+// Format Version: 2
+// Date: Thu Sep 29 13:57:41 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "2301929872 1660332954"
```

Comment:
Indeed, these exclusions were not caught by the UNR tool.

Reply:
I believe I saw the same issue in kmac regarding the step_i :)

Reply:
Should we have an AE investigate these?

Reply:
Is this due to an input port being tied off? That would explain why the UNR tool is not able to find these, because it does not know we tie these off in our testbench. I'm sure there must be a way to feed that information to the tool.

Reply:
This is an input to an internal instance, and it is driven within the design, so it is just an unfortunate tool deficiency that was reported, and I think accepted as a bug.

Reply:
Ok, got it.

Reply:
these look good to me!